packages/website/src/features/Packages/Tabs/InteractTab.tsx

```diff
@@ -38,7 +38,19 @@ export const InteractTab: FC<{
   });
 
   const pathName = usePathname();
-  const activeContract = pathName.split('interact/')[1];
+
+  let activeContractOption: Option | undefined;
```

Comment:
I've worked in other web frameworks in the past where it wasn't necessary to manually split the path information. I'm wondering if that's also the case here?